swagger/v3/api_spec.json

```diff
@@ -5726,1303 +2275,1218 @@
+              "school_id": "24b61d1c-ad95-4000-aee0-afbdd542294a",
+              "delivery_partner_id": "db2fbf67-b7b7-454f-a1b7-0020411e2314"
+            }
+          }
+        }
+      },
+      "ECFPartnershipRequestErrorResponse": {
+        "description": "A list of errors",
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "array",
+            "items": {
+              "anyOf": [
+                {
+                  "$ref": "#/components/schemas/UrnInvalidError"
+                },
+                {
+                  "$ref": "#/components/schemas/PartnershipExistsError"
+                },
+                {
+                  "$ref": "#/components/schemas/SchoolFundingError"
+                },
+                {
+                  "$ref": "#/components/schemas/DeliveryPartnerCohortError"
+                },
+                {
+                  "$ref": "#/components/schemas/OtherProviderRecruitedError"
+                }
+              ]
+            }
+          }
+        }
+      },
+      "ECFPartnershipResponse": {
+        "description": "An ECF partnership",
+        "type": "object",
+        "required": [
+          "data"
+        ],
+        "properties": {
+          "data": {
+            "$ref": "#/components/schemas/ECFPartnership"
+          }
+        }
+      },
+      "ECFPartnershipUpdateAttibutesRequest": {
+        "type": "object",
+        "required": [
+          "delivery_partner_id"
+        ],
+        "properties": {
+          "delivery_partner_id": {
+            "description": "The unique ID of the delivery partner you will work with for this school partnership",
             "type": "string",
             "format": "uuid",
             "example": "cd3a12347-7308-4879-942a-c4a70ced400a"
-          },
-          "uplift_paid": {
-            "description": "If participant is eligible for uplift, whether it has been paid as part of this declaration",
-            "type": "boolean",
-            "example": true
-          },
-          "evidence_held": {
-            "description": "The type of evidence that the lead provider holds on their platform to demonstrate that the participant has met the retention criteria for the current milestone period",
+          }
+        }
+      },
+      "ECFPartnershipUpdateDataRequest": {
+        "description": "Update An ECF partnership",
+        "type": "object",
+        "properties": {
+          "type": {
             "type": "string",
             "enum": [
-              "training-event-attended",
-              "self-study-material-completed",
-              "other"
-            ],
-            "example": "unknown"
-          },
-          "has_passed": {
-            "description": "Whether the participant has failed or passed",
-            "type": "boolean",
-            "example": null
+              "ecf-partnership-update"
+            ]
           },
-          "lead_provider_name": {
```

Comment:
lead provider name seems to be missing from declaration attributes in `ParticipantDeclarationAttributes` in v3, let's watch out for that